### PR TITLE
Adding Talk Links—9 (4.0.4)

### PR DESCRIPTION
**Pull request by mazenchami**
# Description

Now that the talks have been posted, we've updated the `webflow-data.ts` file to include `talk-url` for all available talks.

Version bump is also included here

# Graphics

| iOS            | Android            |
| -------------- | ------------------ |
| <br/> | <br/> |

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

**Review by morganick — CHANGES_REQUESTED** [review]
On Android, the watch button is behind the software buttons:
Screenshot_20230602_101018_Chain React App 2023

**Comment by mazenchami** [comment]
> On Android, the watch button is behind the software buttons: Screenshot_20230602_101018_Chain React App 2023

please pull latest. this commit solves it: https://github.com/infinitered/ChainReactApp2023/pull/238/commits/026e938131bd5383f94953e1cd87b9bba38d7ff9

**Review by morganick — CHANGES_REQUESTED** [review]

File: app/components/Button.tsx [hunk 1]
```diff
@@ -152,14 +153,15 @@ export type FloatingButtonProps = ButtonProps & { isVisible?: boolean }
...
+  const { bottom: paddingBottom } = useSafeAreaInsets()
   return (
     <View>
       {/* <View> must be here as the parent for Layout Animations to work. (entering/exiting animations) */}
       {isVisible ? (
         <Animated.View
           entering={FadeInDown}
           exiting={FadeOutDown}
-          style={[$floatingAction, { width }]}
+          style={[$floatingAction, { bottom: paddingBottom + spacing.extraSmall, width }]}
```
[the remark this answers is not in the document]
Comment (mazenchami) [reply app/components/Button.tsx:164]: fixed here: https://github.com/infinitered/ChainReactApp2023/pull/238/commits/2650a6253db0e134d533e046f007c17ada59b183

[the remark this answers is not in the document]
Comment (mazenchami) [reply app/components/Button.tsx:164]: moved to a `layout` variable here: https://github.com/infinitered/ChainReactApp2023/pull/238/commits/bc73a50ecab36dc550fb1710672ea6fcf7363b20

**Review by morganick — APPROVED** [review]
LGTM 👍

**Comment by morganick** [comment]
@mazenchami one thing I didn't check was the font size changes with the button 🤔

**Comment by morganick** [comment]
@mazenchami never mind, button text size doesn't change based on font size changes to the phone.

**Comment by mazenchami** [comment]
> @mazenchami one thing I didn't check was the font size changes with the button 🤔

we actually disable font scaling on this button
Screenshot 2023-06-02 at 10 50 51 AM

**Pull request merged**